### PR TITLE
Bug/anthropic stream issue

### PR DESCRIPTION
**Pull request by pulphix**
Fix Anthropic Stream

### CODE REVIEW

File: examples/sample_run_local_tools_groq.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
 
 @th.register_local_tool("hello")
-def hello(city: str):
+def hello_tool(city: str):
     """Return Local Time"""
```
Comment (iamdaniele) [inline examples/sample_run_local_tools_groq.py]: Change the comment of the function or remove it so it’s not misleading for the user. The comment says the tool returns a local time, but the function does not

File: examples/sample_run_local_tools_stream.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+@th.register_local_tool("hello")
+def hello_tool(city: str):
+    """Return Local Time"""
+    return f"Hello from {city}!!!"
+
+
+messages: List = [{
+    "role": "user",
+    "content":
+        "Can I get an hello from Rome?"
```
Comment (iamdaniele) [inline examples/sample_run_local_tools_stream.py]: grammar: “Can I get a hello from Rome?”

**Review by iamdaniele — CHANGES_REQUESTED** [review]
Grammar + comments.

In general, when writing prompts and strings, just ask Claude to rewrite them for you to they’re grammatically/syntactically correct

File: examples/sample_run_local_tools_groq.py (hunk 1 above)
Comment (pulphix) [reply examples/sample_run_local_tools_groq.py]: FIXED

File: examples/sample_run_local_tools_stream.py (hunk 2 above)
Comment (pulphix) [reply examples/sample_run_local_tools_stream.py]: FIXED

**Pull request merged**